content/300-guides/300-upgrade-guides/200-upgrading-versions/030-upgrading-to-prisma-4.mdx

```diff
@@ -213,9 +213,10 @@ In the following example of a _one-to-one relation_ between a `User` and `Profil
 
 ```prisma
 model User {
-  id      Int     @id @default(autoincrement())
-  email   String  @unique // <- we enforce this unique attribute
-  profile Profile
+  id        Int     @id @default(autoincrement())
+  email     String  @unique // <- we enforce this unique attribute
+  profile   Profile @relation(fields: [profileId], references: [id])
+  profileId Int
```

Comment:
this might be wrong, but be added because the `Profile` is not shown here which would have to have a forward relation to be able to make `profile Profile` a backrelation (Which is ok not having a `@relation` attribute and scalar field)

Reply:
so we leave as it is? remove `profileId Int`?

Reply:
A docs writer needs to take a bigger look at what the example around this is and then decide what the original intention of this was.

Reply:
@ruheni Would you mind taking a look at this PR and specifically how it affects the Upgrading to Prisma 4 guide?

Reply:
The "baseline" schema that was intended for use in the upgrade path is likely to be (for which the `@unique` attribute is added to the `profileId` scalar field):

```prisma
model User {
  id        Int     @id @default(autoincrement())
  email     String  @unique // <- we enforce this unique attribute
  profile   Profile @relation(fields: [profileId], references: [id])
  profileId Int
}

model Profile {
  id        Int     @id @default(autoincrement())
  userEmail String? @unique
  user      User?
}
```

I'll make a commit updating the "baseline" schema.